apps/catalogue/src/views/TableMappingsView.vue

```diff
@@ -71,13 +71,23 @@
         </td>
       </tr>
       <tr>
+        <th v-if="canEdit"><h6>#</h6></th>
         <th><h6>Target column</h6></th>
         <th><h6>Origin column</h6></th>
         <th><h6>Rule</h6></th>
         <th><h6>Syntax</h6></th>
         <th><h6>Notes</h6></th>
       </tr>
       <tr v-for="(m, index) in variablemappings" :key="index">
+        <td v-if="canEdit">
```

Comment:
Looks still a bit verbose

In styleguide we had https://emx2.dev.molgenis.org/apps/styleguide/#rowbuttonadd and also delete, edit, clone
Why don't we have that in molgenis-components? 

Purpose was to encapsulate the 'isEditModalShown' and 'RowButton' and 'EditModal' in a nice package. 
When I reviews RowButton I must confess I thought that was already the replacement of above button; I didn't realize it was just an empty icon.

Reply:
you can add a wrapper component if needed, not part of this story estimate. Composition instead of duplication allows for more consistent styling and flexible use.

Reply:
see https://github.com/molgenis/molgenis-emx2/pull/1719/files